tasks/main-clevis.yml

```diff
@@ -0,0 +1,45 @@
+---
+- name: Ensure required packages are installed
+  package:
+    name: "{{ __nbde_client_packages }}"
+    state: present
+
+
+- name: Prepare key files, perform clevis operations and dispose of key files
```

Comment:
Is this role idempotent?  If you run the role again with the same `nbde_client_bindings`, will it do all of these steps again, and report that something changed?  If so, I'm not sure how to prevent that from happening - how would you know ahead of time that you have already brought the managed hosts to the desired state?  I realize it may be a little late for this particular PR, and I apologize for not noticing this sooner, so perhaps we can address this in a subsequent PR if necessary.

Reply:
Not fully. We discovered an issue with at least `passphrase_temporary`, which will be addressed after the initial merge.

Reply:
@pcahyna opened #4 to track this issue.